HVZ.Persistence.MongoDB/Repos/GameRepo.cs

```diff
@@ -225,6 +232,109 @@ public async Task<Game> LogTag(string gameId, string taggerUserId, string tagRec
         return newGame;
     }
 
+    public async Task<Game> AddPlayerToOzPool(string gameId, string playerId)
```

Comment:
```suggestion
    public async Task<Game> AddPlayerToOzPool(string gameId, string userId)
```
for clarity as there are many ids a player can have